autodepgraph/graph.py

```diff
@@ -33,7 +33,9 @@ def load_graph(self, filename, load_node_state=False):
                 # If the node does not exist, create a new node
                 node = CalibrationNode(node_snap['name'])
 
-            pars_to_update = ['dependencies', 'check_functions',
+            # children are not set because it is automatically set when
+            # parents are set.
```

Comment:
I would change this comment to make clear that children are set when this node gets added as a parent to another node.